### ISSUE TITLE
Feature suggestion about current installer for EE.

### ISSUE THREAD

**Issue by korn3r** [RedChimera/IWD2EE#114]
Hi.

TL;DR: Please move questions about what to install at the start of installation process.

Longer edition:
I had to install EE many times in last week. I had to reinstall it because i messed up order of installation for localization, then i had to reinstall updated localization, etc.
And i currently have 2-core pentium from mid-2000s.
It installs addon for 1hour+ (closer to 2 hours on this pc).
And my problem with installation is not that its long (althought IT IS LONG...), but the fact i need to be in vicinity of my pc while it happening, because it asks me questions about what to install during process and mostly during second half of the process, and there are at least 4-5 questions. So i have to be close to pc to answer question that pops up after 40 minutes of installation process, then wait for another 10 minutes, then after that another 5-10 minutes until next question...
Yes, pc is old, but i tried it on 4-core AMD A8 and its still ~40 minutes + it shouldnt be that hard to ask those questions at the start, right where it asks if you want to print readme.

Overall ovesome addon. Except those glitchy wyverns in act 3, i hated those.

**Comment by WillScarlettOhara** [comment]
You could test Project Infinity with @Connelly's install order file (available on the Infinity Engine discord, resource channel).
